src/parser.rs

```diff
@@ -313,6 +315,51 @@ pub fn parse_lcov(
     Ok(results)
 }
 
+#[derive(Debug, Default, Deserialize)]
+struct GcovJson {
+    format_version: String,
+    gcc_version: String,
+    current_working_directory: Option<String>,
+    data_file: String,
```

Comment:
What are `current_working_directory` and `data_file` used for?

Reply:
`current_working_directory` is the output of `getpwd()` when the gcno is generated:
https://github.com/gcc-mirror/gcc/blob/e87559d202d90e614315203f38f9aa2f5881d36e/gcc/coverage.c#L1321
`data_file` is the gave as argument of `gcov`

Reply:
Can you add this as a comment to explain the fields?